src/darker/tests/conftest.py

```diff
@@ -14,15 +15,23 @@
 
 
 @pytest.fixture
-def without_isort():
-    with patch.dict(sys.modules, {"isort": None}):
-        yield
-
-
-@pytest.fixture
-def with_isort():
-    with patch.dict(sys.modules, {"isort": types.ModuleType("isort")}):
-        yield
+def isort_present():
```

Comment:
Not sure if it makes sense to wrap this into a fixture, it could just be the context manager

Reply:
Absolutely. Thanks!